test/regression/config.yaml

```
@@ -0,0 +1,6 @@
+lading:
+  version: 0.20.8
+
+target:
+  cpu_allotment: 8
```

```suggestion
  cpu_allotment: 7
```
to match what was previously in `regression_detector.yaml`